docs/pages/deploy-a-cluster/helm-deployments/kubernetes-cluster.mdx

```diff
@@ -12,10 +12,16 @@ Instead, Teleport Cloud users should consult the following guide, which shows yo
 
 Teleport can provide secure, unified access to your Kubernetes clusters. This guide will show you how to:
 
+<ScopedBlock scope={["enterprise"]}>
+- Install the Teleport Enterprise License.
```

Comment:
```suggestion
```
For me, this is more of a step toward deploying Teleport Enterprise on Kubernetes, rather than a standalone step